dracut/30disk-uuid/disk-uuid.service

```diff
@@ -1,18 +1,17 @@
 [Unit]
-Description=Generate new UUID for disk GPT %I
+Description=Generate new UUID for disk GPT if necessary
 DefaultDependencies=no
+
 Wants=local-fs-pre.target
 Before=local-fs-pre.target systemd-fsck-root.service
-Wants=systemd-udevd.service
-After=systemd-udevd.service
-Requires=%i.device
-After=%i.device
+
+Wants=systemd-udevd.service dev-disk-by\x2dlabel-ROOT.device
```

Comment:
```suggestion
Wants=systemd-udevd.service sysroot-usr.mount
```

Reply:
Oh, maybe this is wrong because sysroot-usr.mount maybe only works after the initramfs. Will have a look.